data/json/items/resources/misc.json

```diff
@@ -79,5 +79,31 @@
     "symbol": "=",
     "color": "dark_gray",
     "ammo_type": "components"
+  },
+  {
+    "id": "esapi_plate",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "ESAPI ballistic plate" },
+    "description": "An NIJ IV rated ceramic ballistic plate.",
```

Comment:
can we get a more descriptive term than NIJ IV? I'd suggest writing the description as if a person found it lying on the side of the road and wanted to know what it was for.